board_widgets.go

```diff
@@ -0,0 +1,712 @@
+package datadog
```

Comment:
I know we've been a little inconsistent with it, but could you please add license information like you did for `board.go`?

Reply:
Added, resolving.